lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -77,6 +77,7 @@ function _storeInMDandDeleteData(bucketName, dataGetInfo, cipherBundle,
  * @param {(object|null)} streamingV4Params - if v4 auth, object containing
  * accessKey, signatureFromRequest, region, scopeDate, timestamp, and
  * credentialScope (to be used for streaming v4 auth if applicable)
+ * @param {(object|null)} overheadField - fields to be included in metadata overhead
```

Comment:
(Picking a random line)

Do we also need to support this new parameter in routeBackbeat, where we call the metadata putObjectMD function?
https://github.com/scality/cloudserver/blob/development/7.70/lib/routes/routeBackbeat.js#L554

I believe that we want to keep the same behavior during internal operation as well?

Reply:
Good catch, Backbeat definitely needs to be supported